eval/verification_to_csv.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/python3
+
+# In the results directory
+# Iterate each file named *-verification.json
+# find corresponding file named *.meta.json
+# extract fields from both of them and create a csv row
+# The following columns are available
+# Filename,
+
+
+import argparse
+from csv import writer
```

Comment:
Probably not needed since we already do `import csv`.

Reply:
True!